.github/CODEOWNERS

```diff
@@ -1423,10 +1423,77 @@ x-pack/test/api_integration/deployment_agnostic/services/ @elastic/appex-qa
 x-pack/test/**/deployment_agnostic/ @elastic/appex-qa #temporarily to monitor tests migration
 
 # Core
-/test/plugin_functional/plugins/rendering_plugin @elastic/kibana-core
-/test/plugin_functional/plugins/session_notifications @elastic/kibana-core
+/x-pack/test/functional/page_objects/navigational_search.ts @elastic/kibana-core
+/x-pack/test/stack_functional_integration/apps/savedobjects_upgrade_testing @elastic/kibana-core
+/x-pack/test/functional/page_objects/status_page.ts @elastic/kibana-core
+/x-pack/test/functional/page_objects/share_saved_objects_to_space_page.ts @elastic/kibana-core
+/x-pack/test/functional/page_objects/banners_page.ts @elastic/kibana-core
+/x-pack/test/common/lib/test_data_loader.ts @elastic/kibana-core
+/x-pack/test/api_integration/services/usage_api.ts @elastic/kibana-core
+/x-pack/test/api_integration/apis/kibana @elastic/kibana-core
+/test/api_integration/fixtures/import.ndjson @elastic/kibana-core
+/x-pack/test/plugin_api_integration @elastic/kibana-core # Assigned per https://github.com/elastic/kibana/pull/146704
+/x-pack/test/localization/ @elastic/kibana-core # Assigned per https://github.com/elastic/kibana/pull/146704
+/test/ui_capabilities/newsfeed_err @elastic/kibana-core # Assigned per https://github.com/elastic/kibana/pull/66562
+/test/server_integration/services/types.d.ts @elastic/kibana-core # Assigned per https://github.com/elastic/kibana/pull/81140
+/test/server_integration/http @elastic/kibana-core
+/test/scripts/run_multiple_kibana_nodes.sh @elastic/kibana-core
+/test/plugin_functional/test_suites/application_links @elastic/kibana-core
+/test/interpreter_functional/test_suites/run_pipeline @elastic/kibana-core
+/test/functional/services/usage_collection.ts @elastic/kibana-core
+/test/functional/services/saved_objects_finder.ts @elastic/kibana-core
```

Comment:
`@elastic/appex-sharedux` and I think same for the plugin `src/plugins/saved_objects_finder`

Reply:
Ah this one should be data discovery:
https://github.com/wayneseymour/kibana/blob/assign-shared-ux/src/plugins/saved_objects_finder/kibana.jsonc#L3-L6

Reply:
https://github.com/elastic/kibana/pull/199771/commits/137e6c0dcd16885acbd10ea5eb3f1c9d642dad62